src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainError/EndorsementChainError.test.tsx

```diff
@@ -4,12 +4,24 @@ import { EndorsementChainError } from "./EndorsementChainError";
 
 describe("EndorsementChainError", () => {
   it("should render correctly", () => {
-    render(<EndorsementChainError error={""} />);
+    render(
+      <table>
+        <tbody>
+          <EndorsementChainError error={""} />
+        </tbody>
+      </table>
```

Comment:
i thought this should fix the warning of tr in div but it didn't, wanna double check?

Reply:
i couldn't find the warning anywhere.

Reply:
oh yeah see wrongly didn't pull